arangod/Pregel/PregelFeature.cpp

```diff
@@ -442,3 +443,16 @@ void PregelFeature::handleWorkerRequest(TRI_vocbase_t& vocbase,
     w->aqlResult(outBuilder, withId);
   }
 }
+
+uint64_t PregelFeature::numberOfActiveConductors() const {
+  MUTEX_LOCKER(guard, _mutex);
+  uint64_t nr{0};
+  for (auto const& p : _conductors) {
+    std::shared_ptr<Conductor> c = p.second.second;
```

Comment:
No need to copy the shared_ptr here:
```suggestion
    std::shared_ptr<Conductor> const& c = p.second.second;
```

Reply:
Agreed.